zer0-os/zOS
# feat: update other members 'lastSeenAt' in conversations

### What does this do?
- requests the `getPresence` data and updates the other members (user model) `lastSeenAt `.

### Why are we making this change?
- when fetching the conversations the 'status' tooltip was displaying 'Never' unless a user had fired a presence event.

### How do I test this?
- open messenger and hover over a user in the conversation list > check the tooltip displays the correct 'last seen' time.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

Before:


https://github.com/zer0-os/zOS/assets/39112648/7036e093-5d77-42bb-88ae-3fa2d5cec3b1



After:

https://github.com/zer0-os/zOS/assets/39112648/5ce07ce8-6426-415c-b51b-78c335c2cc9c
